README.md

```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
 * [checkstyle](https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html)
 * [Google java format](https://github.com/google/google-java-format)
 
+* In order to setup this project it's necessary:
+
+  * set BD connexion parameters in `application.properties` (URL, Username, Password)
+  * set BD connexion parameters for testing in route test/resources `applicatin.properties` (URL, Username, Password)
+  * Set oauth2.secret.key in `application.properties` as part of Oauth2 security.
+  * It is important to validate that migrations ran in python side of the project to have BD and tables available.
```

Comment:
I think you should move this to the beginning. This the first thing you need to make sure before even starting the setup process

Reply:
Done.